client/my-sites/stats/activity-log/index.jsx

```diff
@@ -219,11 +225,16 @@ class ActivityLog extends Component {
 			isRewindActive,
 			logs,
 			moment,
+			restoreProgress,
 			siteId,
 			slug,
 			startDate,
 		} = this.props;
 
+		const disableRestore = includes( [
+			'queued',
+			'running',
+		], get( restoreProgress, 'status' ) );
```

Comment:
Just wondering if this might be a bit fragile, for example if another _in progress_ state is added later? A dedicated `isRestoreInProgress()` selector would be no less fragile, but perhaps a more common pattern therefore quicker to diagnose? Up to you.